lib/ctblmono.gi

```diff
@@ -1330,38 +1344,38 @@ InstallMethod( TestMonomial,
 
     Info( InfoMonomial, 1, "TestMonomial called" );
 
-    # Start wirth elementary tests for monomiality.
-    test:= TestMonomialQuick( chi );
+    # Start with elementary tests for monomiality.
+    if   IsIrreducibleCharacter( chi ) then
+      test:= TestMonomialQuick( chi );
+    elif HasIsMonomialCharacter( chi ) then
```

Comment:
Should that `HasIsMonomialCharacter` perhaps be first, given that it'll be the quickest test by far?

Reply:
O.k.